device-types/Nokia/7210-SAS-M-1.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-M-1
+slug: nokia-7210-sas-m-1
+part_number: 3HE04699AAAB01
+u_height: 2
+is_full_depth: false
+subdevice_role: parent
```

Comment:
`subdevice_role: parent` is only to be used if this device is either a parent (i.e. a chassis device that houses other devices), or or a child (i.e. this device slots into a chassis device). Generally parent devices have `device-bays` defined for their children devices.

**If this is not actually a parent device, please remove this definition**

Reply:
Added device-bays

Reply:
Which device does this house?